website/source/docs/providers/azure/r/affinity_group.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "azure"
+page_title: "Azure: azure_affinity_group"
+sidebar_current: "docs-azure-affinity-group"
+description: |-
+    Creates a new affinity group on Azure.
+---
+
+# azure\_affinity\_group
+
+Creates a new affinity group on Azure with its own .cloudapp.net domain.
+
+## Example Usage
+
+```
+resource "azure_affinity_group" "terraform-service" {
+	name = "terraform-group"
+	location = "North Europe"
+    label = "tf-group-01"
+	description = "affinity group created by Terraform."
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the affinity group. Must be unique on your
+    Azure subscription.
+
+* `location` - (Required) The location where the affinity group should be created.
+    For a list of all Azure locations, please consult [this link](http://azure.microsoft.com/en-us/regions/).
+
+* `label` - (Label) A label to be used for tracking purposes. Must be
+    non-void.
```

Comment:
What does non-void mean in this context. Looks like it just means "Required" in which case we can just drop the sentence and change the parens from `(Label)` to `(Required)`